raster/src/main/scala/geotrellis/raster/TileLayout.scala

```diff
@@ -46,7 +45,6 @@ object TileLayout {
  * @param tileCols   number of pixel columns in each tile, East to West
  * @param tileRows   number of pixel rows in each tile, North to South
  */
-@JsonCodec
```

Comment:
Rather than removing codec from the companion object I would prefer deleting codecs from https://github.com/locationtech/geotrellis/blob/7871e015129ae4a1ccfd06a23544a6de9ac68c6e/raster/src/main/scala/geotrellis/raster/io/json/Implicits.scala and https://github.com/locationtech/geotrellis/blob/7eb7958dde643fa4a347c9ac5b055bb333f416fd/spark-pipeline/src/main/scala/geotrellis/spark/pipeline/json/Implicits.scala 

It is just easier to use codecs when they are stored in the companion objects, since you won’t need any extra imports to make circe work.

Can you remove them instead of removing this codecs?